src/app/screens/Auth/screens/Login/styles.ts

```diff
@@ -11,10 +11,49 @@ export default StyleSheet.create({
   form: {
     width: 250
   },
+  input: {
+    borderBottomWidth: 0,
+    backgroundColor: white,
+    width: 300,
+    height: 50,
+    borderRadius: 15,
+    elevation: 1,
+    shadowColor: lightGray,
+    shadowOpacity: 1,
+    shadowOffset: {
+      width: 0,
+      height: 3
+    },
+    alignSelf: 'center',
+    paddingHorizontal: 8
+  },
+  errorContainer: {
+    width: 280,
+    alignSelf: 'center',
+    textAlign: 'left'
+  },
+  inputText: {
+    padding: 10
+  },
+  labelText: {
+    alignSelf: 'center',
+    color: green,
+    width: 290,
+    textAlign: 'left'
+  },
   formButton: {
     backgroundColor: green,
     padding: 10,
-    borderRadius: 3,
-    marginTop: 15
+    borderRadius: 25,
```

Comment:
if values ​​are repeated you could create a variable

for example
COMMON_VALUE_25 = 25

Reply:
added const